piccolo_api/crud/serializers.py

```diff
@@ -141,6 +144,11 @@ def create_pydantic_model(
         }
 
         if isinstance(column, ForeignKey):
+            if nested:
+                _type = create_pydantic_model(
+                    table=column._foreign_key_meta.resolved_references
```

Comment:
I guess this should cal with `nested=True`

Reply:
Yeah, that's right.